app/src/main/java/com/nikunjsondagar/composedemo/navigation/MainScreenNavigationGraph.kt

```diff
@@ -22,28 +25,39 @@ import com.nikunjsondagar.feature_users.presentation.UserListViewModel
 fun MainScreenNavigationGraph(
     navHostController: NavHostController,
     padding: PaddingValues,
-    activity: Activity
+    activity: Activity,
+    sharedViewModel: SharedViewModel
 ) {
+    val userListViewModel = hiltViewModel<UserListViewModel>()
     NavHost(
         navController = navHostController,
         startDestination = NavigationScreens.UserScreen.route,
         modifier = Modifier.padding(padding)
     ) {
         composable(route = NavigationScreens.UserScreen.route) {
-            val userListViewModel = hiltViewModel<UserListViewModel>()
+            sharedViewModel.updateSelectedTab(SharedViewModel.SearchTab.USERS)
```

Comment:
sharedViewModel is not the best idea. To notify two different view models we can use some use case, interface or something like this